docs/tools/lsp-factoryjs/01-getting-started.md

```diff
@@ -28,9 +28,9 @@ const lspFactory = new LSPFactory(provider, {
 });
 ```
 
-## Using LSPFactory in a Dapp
+## Using LSPFactory in a dApp
 
-If being used in the browser in a Dapp, pass the `ethereum` object as the provider parameter to connect to a browser extension like the UniversalProfile Browser extension or MetaMask. This will then prompt users to sign the transactions the LSPFactory deploys smart contracts.
+If used in the browser on a dApp page, pass the ethereum object as the provider parameter to connect to a browser extension like the UniversalProfile browser extension or MetaMask. The wallet application will prompt users to sign the transactions the **LSPFactory** deploys smart contracts.
```

Comment:
Note about "_The wallet application_", the LUKSO browser extension is not a wallet but rather a way for people to interact with their UP. There is nothing stored in the browser extension except the private key which controls the UP. No assets should be stored on that key, assets are stored on the UP itself

```suggestion
If used in the browser on a dApp page, pass the ethereum object as the provider parameter to connect to a browser extension like the UniversalProfile browser extension or MetaMask. The browser extension will prompt users to sign the transactions as the **LSPFactory** deploys the smart contracts.
```